dists/androidsdl/scummvm/AndroidAppSettings.cfg

```diff
@@ -185,7 +185,8 @@ DeleteFilesOnUpgrade="%"
 # Optional shared libraries to compile - removing some of them will save space
 # MP3 support by libMAD is encumbered by patents and libMAD is GPL-ed
 # Available libraries: mad (GPL-ed!) sdl_mixer sdl_image sdl_ttf sdl_net sdl_blitpool sdl_gfx sdl_sound intl xml2 lua jpeg png ogg flac tremor vorbis freetype xerces curl theora fluidsynth lzma lzo2 mikmod openal timidity zzip bzip2 yaml-cpp python boost_date_time boost_filesystem boost_iostreams boost_program_options boost_regex boost_signals boost_system boost_thread glu avcodec avdevice avfilter avformat avresample avutil swscale swresample bzip2 
-CompiledLibraries="mad vorbis flac ogg jpeg png theora freetype faad curl sdl_net"
+#CompiledLibraries="mad vorbis flac ogg jpeg png theora freetype faad curl sdl_net"
+CompiledLibraries="mad vorbis flac ogg jpeg png theora freetype faad"
```

Comment:
This change should be made to `AndroidAppSettings.cfg.in` as well.

Reply:
done.